performance/src/test/com/clickhouse/benchmark/clients/ClientV2.java

```diff
@@ -0,0 +1,103 @@
+package com.clickhouse.benchmark.clients;
+
+import com.clickhouse.client.ClickHouseNode;
+import com.clickhouse.client.api.Client;
+import com.clickhouse.client.api.data_formats.ClickHouseBinaryFormatReader;
+import com.clickhouse.client.api.enums.Protocol;
+import com.clickhouse.client.api.insert.InsertResponse;
+import com.clickhouse.client.api.insert.InsertSettings;
+import com.clickhouse.client.api.query.QueryResponse;
+import com.clickhouse.data.ClickHouseFormat;
+import com.clickhouse.data.ClickHouseOutputStream;
+import com.clickhouse.data.stream.NonBlockingPipedOutputStream;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.clickhouse.client.ClickHouseServerForTest.isCloud;
+
+@State(Scope.Thread)
+public class ClientV2 extends BenchmarkBase {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClientV2.class);
+    Client client;
+
+    @Setup(Level.Trial)
+    public void setup(DataState dataState) throws Exception {
+        super.setup(dataState);
+    }
+
+    @Setup(Level.Iteration)
+    public void setUpIteration() {
+        LOGGER.info("Setup Each Invocation");
+
+        ClickHouseNode node = getServer();
+        client = new Client.Builder()
+                .addEndpoint(Protocol.HTTP, node.getHost(), node.getPort(), isCloud())
+                .setUsername(getUsername())
+                .setPassword(getPassword())
+                .compressClientRequest(true)
+                .setMaxRetries(0)
+                .setLZ4UncompressedBufferSize(2 * 1024 * 1024)
+                .setClientNetworkBufferSize(20 * 1024 * 1024)
+                .setSocketRcvbuf(2 * 1024 * 1024)
```

Comment:
it's not a default experience. How many users know what parameters should be adjusted and how?